### PR TITLE
Removes a useless 4000 item typecache

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
We don't need this, we definitely do not need to keep it around from world init, it can go.

### CODE REVIEW

File: code/modules/awaymissions/capture_the_flag.dm [hunk 1]
```diff
@@ -342,12 +331,20 @@
 
 /obj/machinery/capture_the_flag/proc/reset_the_arena()
 	var/area/A = get_area(src)
+	var/list/ctf_object_typecache = typecacheof(list(
```
Comment (nicbn) [inline code/modules/awaymissions/capture_the_flag.dm]: Make it global

**Review by nicbn — CHANGES_REQUESTED** [review]

File: code/modules/awaymissions/capture_the_flag.dm (hunk 1 above)
Comment (vuonojenmustaturska) [reply code/modules/awaymissions/capture_the_flag.dm]: it doesn't need to exist after the proc is over

**Review by SpaceManiac — APPROVED** [review]

File: code/modules/awaymissions/capture_the_flag.dm (hunk 1 above)
Comment (nicbn) [reply code/modules/awaymissions/capture_the_flag.dm]: Make it global and only initialize it when proc is called, but keep it stored so it won't be initialized again if the proc is called again.

Comment (SpaceManiac) [reply code/modules/awaymissions/capture_the_flag.dm]: CTF almost never happens so I'm not worried about this really, but feel free to PR the tweak yourself

Comment (vuonojenmustaturska) [reply code/modules/awaymissions/capture_the_flag.dm]: it doesn't need to exist, we don't have to pay a permanent cost of this for something that barely ever happens and doesn't cost all that much to generate when needed

**Pull request merged**